src/components/PullRequestList/index.tsx

```diff
@@ -18,90 +17,87 @@ type PullRequestListProps = {
     prType: string;
 };
 
+function ScrollTop() {
+  const topMargin = (document.documentElement
+    && document.documentElement.scrollTop)
+    || document.body.scrollTop;
+  return topMargin;
+}
+function ScrollHeight() {
+  const WindowHeight = (document.documentElement
+    && document.documentElement.scrollHeight)
+    || document.body.scrollHeight;
+  return WindowHeight;
+}
+
 const PullRequestList: FC<PullRequestListProps> = ({ prType }) => {
   const [pullRequests, setPullRequests] = useState<pullRequestType[]>([]);
-  const [loading, setLoading] = useState(false);
-  const [error, setError] = useState('');
+  const [noData, setNoData] = useState(false);
   const [page, setPage] = useState(1);
+  const [isBottom, setIsBottom] = useState(false);
 
   const prUrl = `${process.env.NEXT_PUBLIC_BASE_URL}/pullrequests/${prType}?page=${page}`;
+  const {
+    response,
+    error,
+    isLoading,
+  } = useFetch(prUrl);
 
-  const fetchPRs = async () => {
-    try {
-      setLoading(true);
-      const response = await axios.get(prUrl);
-      setPullRequests(response.data.pullRequests);
-
-      if (!response.data.pullRequests.length) {
-        setError(`No more ${prType} PRs...`);
-      } else {
-        setError('');
-      }
-    } catch (err) {
-      setPullRequests([]);
-      setError('Error fetching pull requests!');
-    } finally {
-      setLoading(false);
+  useEffect(() => {
+    if (!response?.pullRequests?.length && page > 1) {
+      setNoData(true);
     }
-  };
-
-  const handlePrev = () => {
-    setPage(page > 1 ? page - 1 : 1);
-  };
-
-  const handleNext = () => {
-    setPage(pullRequests.length ? page + 1 : page);
-  };
+    if (response?.pullRequests?.length > 0) {
+      if (page > 1) {
+        setPullRequests((pullRequest) => [...pullRequest, ...response.pullRequests]);
+      } else { setPullRequests(response.pullRequests); }
```

Comment:
Please indent the `else` block consistently 😄

Reply:
No we don't I will change it, Thanks